assets/css/app.css

```diff
@@ -0,0 +1,68 @@
+body {
+  font-family: 'Poiret One', cursive;
+  font-size 1.6em
+}
+
+.form__button--submit {
+  width: 15%;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  margin-bottom: 0.5em;
+}
+
+.filters__button {
+  text-align: center
+}
+
+.filters__button--active   {
+  width: 15%;
+}
+
+.filters__button--all {
+  width: 15%;
+}
+
+.filters__button--complete {
+  width: 15%;
+}
```

Comment:
If you want to put Bootstrap 'inside' things, use mixins with Sass. These are an alternative way to mix in some parts of Bootstrap without using Bootstrappy class names.

Foundation has Sass mixins too, as does Bourbon, Neat, and Refills.